rpc/debug/debug.proto

```diff
@@ -53,8 +53,7 @@ message DebugConfigReq {
     string sketch_path = 3;
     string port = 4;
     string interpreter = 5;
-    bool verbose = 6;
-    string import_file = 7;
+    string import_file = 6;
```

Comment:
It's ok to remove a field but do not change the numeric identifier of existing fields like in:

```diff
-    string import_file = 7;
+￼   string import_file = 6;
```

this provides a better backward compatibility (well, at least the "verbose" value won't go into "import_file" when using an old client). I'll fixi it while rebasing #687

Reply:
Thanks for the information. I was following https://github.com/arduino/arduino-cli/pull/689 as a reference, but without really knowing what I was doing.